docs/recipes/auth.md

```diff
@@ -1,81 +1,57 @@
 # Auth Recipes
 
+Auth recipes assume that you have passed the following config when creating your store:
+
+```js
+const rrfConfig = {
+  userProfile: 'users', // can be any string path
+}
+```
+
 ## Google Login
 
-Here is an example of a component that shows a Google login button if the user is not logged in, and a welcome message if they are. The initial loading state is handled with a simple "loading" message
+Here is an example of a component that shows a Google login button if the user is not logged in, and a welcome message if they are. The initial loading state is handled with a simple "Loading..." message:
 
 ```js
-import React, { Component } from 'react'
+import React from 'react'
 import PropTypes from 'prop-types'
-import GoogleButton from 'react-google-button'
+import { compose } from 'redux'
 import { connect } from 'react-redux'
-import {
-  firebaseConnect,
-  isLoaded,
-  isEmpty
-} from 'react-redux-firebase'
-
-@firebaseConnect() // add this.props.firebase
-@connect( // map redux state to props
-  ({ firebase: { authError } }) => ({
-    authError
-  })
+import { firebaseConnect, isLoaded, isEmpty } from 'react-redux-firebase'
+// import GoogleButton from 'react-google-button' // optional
+
+export const LoginPage = ({ firebase, auth }) => (
+  <div className={classes.container}>
+    <button // <GoogleButton/> button can be used instead
+      onClick={() => firebase.login({ provider: 'google', type: 'popup' })}
+    >Login With Google</button>
+    <div>
+      <h2>Auth</h2>
+      {
+        isLoaded(auth)
```

Comment:
@prescottprue I think it should be !isLoaded(auth)

Reply:
Yes it should. I'll change in an upcoming release, or you can feel free to open an issue.